src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/DataEntry/DataEntry.component.js

```diff
@@ -424,7 +424,7 @@ const WrappedDataEntry = withDataEntryField(buildCompleteFieldSettingsFn())(Save
 type Props = {
     formFoundation: RenderFoundation,
     programName: string,
-    orgUnitName: string,
+    orgUnit: Object,
```

Comment:
Hey,  
If I am not mistaken we know the shape of the `orgUnit` object. It is something like this: 
```
type OrgUnit {
      id: string, 
      name: string, 
      code: string
}  
```
Therefore, can you use a type inference instead of `orgUnit: Object` throughout all the files in the PR?
Thank you!